src/lib/Components/Consignments/Screens/FinalSubmissionQuestions.tsx

```diff
@@ -20,14 +22,18 @@ interface Props extends ViewProperties {
   setup: ConsignmentSetup
 }
 
+const Loader = (p, c) => <Spinner style={{ flex: 1 }} />
```

Comment:
This is temporary, this might not be needed considering the final submission is just a flag for publish instead of draft